### DISCUSSION TITLE
Guidance on floating Command Bar

### DISCUSSION THREAD

**Discussion by synnottd** [twilio-labs/paste#1094] [General] [answered]
Hi,

I was wondering if the paste team had any guidance on a floating command bar for console. Previously we've using the react-components one. But when I looked through the roadmap I couldn't see it, so I was wondering if I was missing something. I've enjoyed not writing CSS (<3 paste) for a year but I'd be happy to implement the floating part if that's the suggestion (using box and button for the actual command bar).

cc: @pguptaDS

**Comment by SiTaggart** [comment, marked as answer]
Hi @synnottd 

That is a great question, for which we don't have a great answer yet.

Right now, we don't really have current, up to date research to suggest that the command bar is something we would want to continue investing in, or stop using, either way. This especially relates to if it's something that is useful to _all_ Twilio products, outside of console.

If that is the case, it might not be something we build at all. It could end up simply being documented as a pattern. We're just not sure right now.

In the meantime, feel free to compose this using Box and Button if the team feels like it's the right design decision for the product or feature, as we don't have a timeline to say either way what we'll be doing for it.

**Comment by synnottd** [comment]
Thanks for the quick response @SiTaggart. This all makes sense. I guess it might become a bit clearer when the edit resource pattern is documented.
